documents/src/pages/elements/text-field.md

```diff
@@ -48,9 +48,118 @@ p {
 
 ::
 
-`ef-text-field` is a form element for text.
+`ef-text-field` is a form element for text. This basic text input enables the user to type a small amount of text and submit it into Workspace.
 
-## Usage
+## Design Usage Rules
+
+### Content principles
+
+* Users should understand the state of the field quickly and easily
+* Placeholder text should not contain instructions. If these are needed supportive text should be provided
+* Label text can wrap onto multiple lines if needed. Avoid truncating, where possible  
+* Supportive  text should be used only where needed
+
+### Formatting
+
+Form labels should be visible and there are 3 variants:
+
+#### Stacked
+
+Stacked is the default form label location
+
+::
+
+```javascript
+::text-field::
+```
+
+```css
+label {
+  display: block;
+}
+ef-text-field {
+  width: 240px;
+}
+```
+
+```html
+<label id="name">Full Name</label>
+<ef-text-field 
+  aria-labelledby="name"
+  placeholder="Your name as shown on your passport">
+</ef-text-field>
+```
+
+::
+
+#### Left Label
```

Comment:
- Update Sandbox examples in 'Formating' to match the Figma component styling and spacing https://www.figma.com/file/YeKQAvy9EGQyggtae5tvqw/HALO---LIGHT----UI-COMPONENT-LIBRARY?node-id=22890%3A6832&t=53RHf2i3CPQdjW74-4
- Remove placeholder text from the 'Formatting' examples and add it as supportive text (to match the Figma asset)